docs/src/main/asciidoc/mp/integrations/oci.adoc

```diff
@@ -29,80 +29,233 @@ include::{rootdir}/includes/mp.adoc[]
 
 - <<Overview, Overview>>
 - <<maven-coordinates, Maven Coordinates>>
-- <<Usage, Usage>>
-- <<Examples, Examples>>
+- <<_accessing_oci_services, Accessing OCI Services>>
 - <<References, References>>
 
 == Overview
 
-Helidon MP OCI Integration provides easy access to Oracle Cloud Infrastructure using the OCI Java SDK.
+You can quickly and easily deploy Helidon MP applications on Oracle Cloud Infrastructure (OCI) and integrate them with OCI services using the OCI Java SDK and the Helidon OCI SDK Extension.
+
+link:{oci-javasdk-url}[The Oracle Cloud Infrastructure SDK for Java] enables you to write code to manage Oracle Cloud Infrastructure resources. The Helidon OCI SDK link:{jakarta-cdi-spec-url}#spi[CDI portable extension] provides support for
+injecting link:{oci-javasdk-url}[Oracle Cloud Infrastructure SDK Clients] in your Helidon MicroProfile application.
 
 include::{rootdir}/includes/dependencies.adoc[]
 
 [source,xml]
+.Adding the Helidon OCI SDK Extension dependency
 ----
 <dependency>
      <groupId>io.helidon.integrations.oci.sdk</groupId>
      <artifactId>helidon-integrations-oci-sdk-cdi</artifactId>
 </dependency>
 ----
 
-== Usage
+NOTE: If your Helidon application is based on the Helidon MP OCI project starter, many of the following configurations are already enabled and you do not need to configure them again. See xref:../guides/oci-guide.adoc[OCI Oracle Cloud Infrastructure Tutorial] for more details.
 
-When added to your application Helidon OCI SDK link:{jakarta-cdi-spec-url}#spi[CDI portable extension] provides support for
-injecting link:{oci-javasdk-url}[Oracle Cloud Infrastructure SDK Clients] in your Helidon MicroProfile application.
+== Authentication
+
+You must configure authentication between your local environment and the OCI environment. It is recommended that you configure authentication first and then verify your configuration by using the link:{oci-javasdk-url}[OCI CLI] to access the service.
+
+The Helidon OCI SDK extension authenticates with OCI by picking up OCI credentials from your environment.
+
+To configure authentication, add the ``oci.auth-strategy`` property to ``/src/main/resources/META-INF/microprofile-config.properties``
 
-The extension also handles authenticating with OCI by automatically picking up OCI credentials from your environment.
+The ``oci.auth-strategy`` property specifies the OCI client authentication mechanism that should be used. It can be a single value or a list of authentication types, separated by commas. If you specify a list, it cycles through each type until the authentication is successful.
```

Comment:
Is it oci.auth-strategy or oci.auth-strategies as in the javadocs? https://helidon.io/docs/v4/apidocs/io.helidon.integrations.oci.sdk.cdi/io/helidon/integrations/oci/sdk/cdi/OciExtension.html

Reply:
@klustria ?

Reply:
Either should work, maybe just go with the plural